main.cpp

```diff
@@ -10,158 +10,162 @@ video: Chapter 2 - Part 3
+    
+9) in the main function at the end: 
+    for each of those functions declared, 
+        a) write out how the function would look if called with correct arguments
+        b) if the function returned anything, store it in a local variable via the 'auto' keyword.
+        c) pass the local variables to ignoreUnused() as you did in 2b)
+    see main() for an example of this.
+ 
+10) click the [run] button.  Clear up any errors or warnings as best you can.
+ */
+
+//2)
+
+    void variableDeclarations()
+    {
+
+        int shoes = 33;
+        int violins = 40;
+        int ports = 2;
+
+        float height = 33.3f;
+        float weight = 94.8f;
+        float temperature = 98.5f;
+
+        bool areOpen = false;
+        bool canBeRemoved = true;
+        bool isVisible = false;
+
+        double bagels = 6004; 
+        double creamCheese = 884403;
+        double coffeeCups = 393939;
+
+        char grade = 'B';
+        char knob = 'H';
+        char view = 's';
+
+        unsigned int blackPants = 7;
+        unsigned int greenJackets = 5;
+        unsigned int shoulderPads = 567;
+
+        ignoreUnused(shoes, violins, ports, height, weight, temperature, areOpen, canBeRemoved, isVisible, bagels, creamCheese, coffeeCups, grade,         knob, view, blackPants,greenJackets, shoulderPads);
+
+    }
+
+/*
+ 10 functions
+ example:
+ note: this example shows the result after completing steps 3-8
+ */
+
+
+int countClothing(unsigned int blackPants, unsigned int greenJackets, unsigned int shoulderPads) { ignoreUnused(blackPants, greenJackets, shoulderPads);
+return { }; }
+
+float eatFood(double bagels, double creamCheese) { ignoreUnused(bagels, creamCheese);
+return { }; }
+
+bool unmuteChannel(int volume, bool isMuted) { ignoreUnused(volume, isMuted);
+return { }; }
+
+float calculateStatus(float weight, float height, float temperature, int active, int orientation) { ignoreUnused(weight, height, temperature, active, orientation);
+return { }; }
+
+double getNumberOfSamples (float getSamples){ ignoreUnused(getSamples);
+return { }; }
+
+int makePhoneCall (int phoneNumber, int areaCode, int countryCode, int unlockPhoneCode){ ignoreUnused(phoneNumber, areaCode, countryCode, unlockPhoneCode);
+return { }; }
+
+char getAddress(int streetNumber, int suiteNumber, char streetName, char nameOfCity, char nameOfState, int zipCode){ ignoreUnused(streetNumber, suiteNumber, streetName, nameOfCity, nameOfState, zipCode);
+return { }; }
+
+int mailLetter(int envelope, float stamp, int mailbox, int letter){ ignoreUnused(envelope, stamp, mailbox, letter);
+return { }; }
+                                                                   
+double buyStamps(bool postOfficeOpen){ ignoreUnused(postOfficeOpen);
+return { }; }
+
+int cookDinner(int oven, int airFryer, int chicken, int brownRice, int chocolateCake, bool isHungry){ ignoreUnused(oven, airFryer, chicken, brownRice, chocolateCake, isHungry);
+return { }; }
```

Comment:
culy braces go on their own line with no other code
```
int funcName(int a, float b);
{
    ignoreUnused(a, b);
    return { };
}
```
- [ ] revise, conform to the course coding standard

Reply:
Completed to the point that it runs, but don't understand why to remove the ";" at the end of the lines where I declare the variables. For example: 

_float calculateStatus(float weight, float height, float temperature, int active, int **orientation)**
{
    ignoreUnused(weight, height, temperature, active, orientation);
    return { };
}_

 -- semicolon removed after "**orientation)**" and others. In your example above, the semicolon is present.